cloud_snitch/lock.py

```diff
@@ -108,6 +100,6 @@ def lock_environment(driver, account_number, name):
         if lock.locked:
             lock.release()
             logger.debug("Released lock on {}: {}".format(
-                lock.account_number,
-                lock.name)
+                lock.env.account_number,
+                lock.env.name)
```

Comment:
should this be printing the uuid instead?
mostly asking because places like cloud_snitch/terminate.py and cloud_snitch/exc.py we're printing uuid's now instead of the account number/name.